### DISCUSSION TITLE
how do i clear openshells icon cache (if it has one)

### DISCUSSION THREAD

**Discussion by samiscoolest** [Open-Shell/Open-Shell-Menu#1870] [Q&A] [answered]
how do i clear openshells icon cache, not that im 100% sure it has one, but it seems like it does
basically i mass replaced all icons in windows 10 ltsc upon installing it using a program that does this, this applied to every program no problem. while installing drivers i ticked some update that made all the icons get overwritten back to the default 10 ones, after once again replacing them, it applied fine to all programs, except openshell, i tried just uninstalling it and reinstalling but that just does nothing, as openshell doesnt seem to uninstall itself very well. anyone know what to do in this situation, and if there is an icon cache of some sort causing this, where it is? it seems like this would be the case since the icons its using should not be accesible anymore and when you view what icon its using in the settings, it shows what icon it should have but its not using it. and yes im aware i could just alter the icons within openshell, but that doesnt work for the folder icons in the "programs" dropdown to my knowledge

**Comment by ge0rdi** [comment, marked as answer]
`Open-Shell Menu Settings` -> `Backup` -> `Clear cached information`
